test_stack.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+"""Test file for 'stack.py'
+
+Run this file with pytest to test conditions
+"""
+from __future__ import unicode_literals
+import pytest
+import stack
+
+
+value = [1, 2, 3, 4, 5]
+foo = stack.Stack(value)
```

Comment:
These should be in a fixture, not in the module namespace.

Up here, they are subject to being mutated by the tests you are about to run.  This means that what one tests does to foo effects what other tests will expect `foo` to look like.  That's brittle and dangerous.  The purpose of fixtures is to provide a clean, well-known world for each and every test to run inside. You want that, use it.